webservices/common/models/costs.py

```diff
@@ -69,9 +74,5 @@ def pdf_url(self):
         return utils.make_report_pdf_url(self.beginning_image_number)
```

Comment:
The EC view has both the Form 9 beginning image number (f9_begin_image_num) and the SB (transaction) image number (sb_image_num)